homeassistant/components/emoncms/config_flow.py

```diff
@@ -204,6 +210,7 @@ async def async_step_init(
                     vol.Required(
                         CONF_ONLY_INCLUDE_FEEDID, default=include_only_feeds
                     ): selector({"select": dropdown}),
+                    vol.Optional(CONF_SYNC_ALL_FEEDS): cv.boolean,
```

Comment:
I think this is weird UX. One can select specific feeds, while also able to check the box to sync all feeds?

Reply:
> So I worked on a better one, with a selector to choose the synchronization mode (manual or auto) added to the form where the user fill in the entry parameters (URL, API_KEY)

This needs to be manual by default with a clear warning that if you have a high number of Feeds in emoncms, that could be problematic! I'd add a check for number of Feeds available and set a max threshold before a second warning.

Reply:
yes it is manual by default.